codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/AwsJson.kt

```diff
@@ -41,6 +42,7 @@ sealed class AwsJsonVersion {
 class AwsJsonHttpBindingResolver(
     private val model: Model,
     private val awsJsonVersion: AwsJsonVersion,
+    private val todoHandlingInitialMessages: Boolean,
```

Comment:
maybe leave a comment on why this flag exists and when it can be removed

Reply:
Addressed in 96ae6b8.